app/templates/base-en.html

```diff
@@ -97,12 +97,13 @@
 } -%}
 
 {%- block preHeader -%}
+    {% set cookiesPageURL = domain_url_en + '/en/cookies/' %}
 
     {%- from 'components/cookies-banner/_macro.njk' import onsCookiesBanner -%}
     {{ onsCookiesBanner({
-            "statementTitle": 'Tell us whether you accept cookies',
-            "statementText": 'We use <a href="' + domain_url_en + '/en/cookies/' + '">cookies to collect information</a> about how you use start.surveys.ons.gov.uk. We use this information to make the website work as well as possible and improve our services.',
-            "confirmationText": 'You’ve accepted all cookies. You can <a href="' + domain_url_en + '/en/cookies/' + '">change your cookie preferences</a> at any time.',
-            "secondaryButtonUrl": domain_url_en + 'en/cookies/'
+        'lang': 'en',
+        'settingsLinkURL': cookiesPageURL,
+        'statementText': '<p>Cookies are small files stored on your device when you visit a website. We use some essential cookies to make this website work.</p><p>We would like to set <a href="' + cookiesPageURL + '">additional cookies</a> to remember your settings and understand how you use the site. This helps us to improve our services. </p>',
+        'preferencesText':  'You can <a href="' + cookiesPageURL + '">change your cookie preferences</a> at any time.'
```

Comment:
Had a chat about it, this is the text from the design system which we would like to be using by default but can't because of a bug in how it passes through the link, so all good 👍